pandas/tests/arrays/string_/test_string.py

```diff
@@ -347,10 +397,6 @@ def test_comparison_methods_array(comparison_op, dtype):
         expected = pd.array(expected, dtype=expected_dtype)
         tm.assert_extension_array_equal(result, expected)
 
-        result = getattr(a, op_name)(pd.NA)
-        expected = pd.array([None, None, None], dtype=expected_dtype)
-        tm.assert_extension_array_equal(result, expected)
```

Comment:
For this case of comparing with NA, we already have a dedicated test just above, so removing it here